Readme.md

```diff
@@ -463,7 +532,80 @@ Not documented yet
 
 ### HDFS dump for batch processing
 
-To come in a next release
+Used to flush metrics per events and days on hdfs parquet table.
+
+It is usefull for long term retention like capacity planning requests.
+
+#### Install Garmadon HDFS reader for batch processing
+
+Garmadon-readers-hdfs's jar can be fetch from maven central:
+```
+<dependency>
+  <groupId>com.criteo.java</groupId>
+  <artifactId>garmadon-readers-hdfs</artifactId>
+  <version>0.0.1</version>
+</dependency>
+```
+
+#### Configure Garmadon HDFS reader
+
+1. Create a configuration directory of your choice
+2. Create a __garmadon-config.yml__ file in this directory
+ 
+ Below is a recommended example:
+ ```
+ hdfs:
+   finalDir: /tmp/hdfs-exporter/final           # Hdfs folder where final parquet files are moved
+   baseTemporaryDir: /tmp/hdfs-exporter/temp    # Temporary hdfs folder where parquet file are created and append
+   messagesBeforeExpiringWriters: 500           # OPTIONAL: Soft limit (see 'expirerPeriod') for number of messages before writing final files (DEFAULT: 3 000 000)
+   writersExpirationDelay: 1                    # OPTIONAL: Soft limit (see 'expirerPeriod') for time since opening before writing final files (in minutes) (DEFAULT: 30)
```

Comment:
is it normal that our recommended values are all at 1 and we do not use the default setting ?